.circleci/config.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           muffet \
             -e 'https://github\.com/runatlantis/atlantis/edit/master/.*' \
             -e 'https://github.com/runatlantis/helm-charts#customization' \
+            -e 'https://github.com/runatlantis/helm-charts/blob/main/README.md#customization' \
```

Comment:
Just nit. Instead of excluding another URL explicitly, would it be better to either
1) Change where `https://github.com/runatlantis/helm-charts/blob/main/README.md#customization` was introduced ([here](https://github.com/runatlantis/atlantis/blob/master/runatlantis.io/docs/using-slack-hooks.md?plain=1#L47)) into `https://github.com/runatlantis/helm-charts#customization` since both URLs actually point to the same doc.
2) Make the argument a pattern, like `https://github.com/runatlantis/helm-charts.*`

I personally favor 1) slightly.

Reply:
agree, we should probably just consolidate the README refs

Reply:
@lilincmu Thank you for reviewing.
I fixed markdown url instead of adding exclude settings in [53ea788](https://github.com/runatlantis/atlantis/commit/53ea788d730f5748587f2ea793c9783ba162fbe4).

Reply:
Thank you too! 😄